README.md

```diff
@@ -113,6 +113,30 @@ We have standardized on a few abbreviations that are allowed to use:
 - `ui` for user interface
 - `util` or `utils` for utility or utilities, as in `StringUtils`
 
+```js
+// bad
+onPrevClick();
+
+// good
+onPreviousClick();
```

Comment:
An event handler should not be called directly. So we maybe we can change this to:
```suggestion
```js
// bad
prevButton.addEventListener('click', onPrevClick);

// good
previousButton.addEventListener('click', onPreviousClick);
```